pkg/kubelet/prober/prober_test.go

```diff
@@ -333,3 +333,138 @@ func TestNewExecInContainer(t *testing.T) {
 		}
 	}
 }
+
+func TestNewProber(t *testing.T) {
+	runner := &containertest.FakeContainerCommandRunner{}
+	recorder := &record.FakeRecorder{}
+	prober := newProber(runner, recorder)
+
+	tests := []struct {
+		name   string
+		got    interface{}
+		expect interface{}
+	}{
+		{"non-nil prober", prober != nil, true},
+		{"runner set", prober.runner, runner},
+		{"recorder set", prober.recorder, recorder},
+		{"exec probe initialized", prober.exec != nil, true},
+		{"http probe initialized", prober.http != nil, true},
+		{"tcp probe initialized", prober.tcp != nil, true},
+		{"grpc probe initialized", prober.grpc != nil, true},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if tt.got != tt.expect {
```

Comment:
Please use either `assert.Equal` or `require.Equal` for more readability and consistency, thanks.

Reply:
I expected that test cases would include some parameters and tested function (`newProber`) would be called for every test case. Calling it only once makes me think that it would be easier to do it without `tests`, e.g.
```
t.Run(tt.name, func(t *testing.T) {
        runner := &containertest.FakeContainerCommandRunner{}
	recorder := &record.FakeRecorder{}
	
       prober := newProber(runner, recorder)
       assert.NotNil(prober)
       assert.Equal(runner, prober.runner)
       ....
}
```

Reply:
Hopefully this one has been resolved in the most recent changes.